### PR TITLE
[FE] Feat: Policy-based Alert details page

### LINKED ISSUE #2138: Create Policy-based Alert Details page
### Description

Currently we have an alert details page that relates to alerts generated through rules. These alerts have events, alert summarizations, prefetches etc. that don't match what a policy-based alert has

We should create a separate view for the policy-based alert

### Related Services

web

### Designs

https://app.abstract.com/share/4fac9036-b9c3-41bb-af26-1d5698df4203

### Acceptance Criteria

- Disable any un-unneeded prefetching 
- Render proper view depending on alert type
- Render proper data for policy-based alert
- Abide to designs
- Test that the correct view & data get rendered for each alert type

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

This PR is the final piece in the unified alerts epic, adding an alert details page that's specifically tailored to policy-based alerts.

Closes #2138 

## Screenshots

Screen Shot 2020-12-10 at 7 21 28 PM

## Designs

https://app.abstract.com/projects/558a8de6-7134-4c8b-91c5-62074bb1279b/branches/df9d59c8-eeb1-452d-a5f5-3712b64010be/collections/1392f601-59e0-4e01-8ca0-fbd3edc9bda2

## Changes

- List your changes here in more detail

## Testing

- `npm run test`

### CODE REVIEW

File: web/src/pages/AlertDetails/AlertDetails.tsx [hunk 1]
```diff
@@ -18,49 +18,22 @@
 
 import React from 'react';
```
Comment (3nvi) [inline web/src/pages/AlertDetails/AlertDetails.tsx:19]: Alert Detaills page now performs a single `getAlert` and based on the alert's `type`, decides what to render

File: web/src/pages/AlertDetails/PolicyAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/**
```
Comment (3nvi) [inline web/src/pages/AlertDetails/PolicyAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx:1]: we now have:

- AlertDetails page (a single file that fetches the alert)
  - PolicyAlertDetails sub-page (with its own components)
  - RuleAlertDetails sub-page (with its own components)

File: web/src/pages/AlertDetails/PolicyAlertDetails/AlertDetailsInfo/AlertDetailsInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
```
Comment (3nvi) [inline web/src/pages/AlertDetails/PolicyAlertDetails/AlertDetailsInfo/AlertDetailsInfo.tsx:2]: check out the path in each file. It's important so you know if this file relates to "rule" or "policy"

File: web/src/pages/AlertDetails/RuleAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx [hunk 4]
```diff
@@ -43,21 +43,20 @@ const AlertDetailsBanner: React.FC<AlertDetailsBannerProps> = ({ alert }) => {
           {alert.title || alert.alertId}
         </Heading>
         <Flex spacing={2} as="ul" flexShrink={0} ml="auto">
-          <Box as="li" aria-describedby="alert-severity-description">
```
Comment (3nvi) [inline web/src/pages/AlertDetails/RuleAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx:46]: all these `aria-describedby` were un-used

File: web/src/pages/AlertDetails/RuleAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx [hunk 5]
```diff
@@ -66,24 +65,22 @@ const AlertDetailsBanner: React.FC<AlertDetailsBannerProps> = ({ alert }) => {
           </Box>
         </Flex>
         <Flex>
-          <Box color="navyblue-100" aria-describedby="alert-id" as="dd" pr={2}>
```
Comment (3nvi) [inline web/src/pages/AlertDetails/RuleAlertDetails/AlertDetailsBanner/AlertDetailsBanner.tsx:69]: `aria-describedby` is not needed in `<dd>` <-> `<dl>` <-> `<dt>` items

File: web/src/pages/AlertDetails/RuleAlertDetails/RuleAlertDetails.tsx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
```
Comment (3nvi) [inline web/src/pages/AlertDetails/RuleAlertDetails/RuleAlertDetails.tsx:19]: copy-paste from the old `AlertDetails` page

File: web/src/pages/AlertDetails/PolicyAlertDetails/PolicyAlertDetails.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    );
+  }
+
+  return (
+    <Box as="article" mb={6}>
+      <Flex direction="column" spacing={6}>
+        <AlertDetailsBanner alert={alert} />
+        <Card position="relative" p={6}>
+          <ErrorBoundary>
+            <AlertDetailsInfo alert={alert} policy={data?.policy} />
```
Comment (alexmylonas) [inline web/src/pages/AlertDetails/PolicyAlertDetails/PolicyAlertDetails.tsx:65]: super optional: `data` can't be undefined at this point.

**Review by alexmylonas — APPROVED** [review]
Solid work @3nvi! I am happily approving. 🚢

File: web/src/pages/AlertDetails/PolicyAlertDetails/PolicyAlertDetails.tsx (hunk 7 above)
Comment (3nvi) [reply web/src/pages/AlertDetails/PolicyAlertDetails/PolicyAlertDetails.tsx:65]: ah you are right

**Pull request merged**